app/src/main/java/com/dcrandroid/dialog/send/SendDialog.kt

```diff
@@ -124,9 +128,84 @@ class SendDialog(val fragmentActivity: FragmentActivity, dismissListener: Dialog
         clearEstimates()
     }
 
+    override fun onPause() {
+        super.onPause()
+        savedInstanceState = Bundle()
+        if (savedInstanceState != null) {
```

Comment:
`savedInstanceState` has been initialized in the previous line so this null check is not needed.